### PR TITLE
Remove warning about protobuf 3.0.0+ from the Makefile

### PR DESCRIPTION
**Pull request by murgatroid99**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2285,28 +2278,6 @@ install-certs: etc/roots.pem
 	$(Q) $(INSTALL) -d $(prefix)/share/grpc
 	$(Q) $(INSTALL) etc/roots.pem $(prefix)/share/grpc/roots.pem
 
-verify-install:
```
Comment (nicolasnoble) [inline Makefile]: You removed the "verify-install" target, but forgot to remove the dependency on that target later.

**Comment by murgatroid99** [comment]
test this please

**Comment by murgatroid99** [comment]
test this please

**Comment by murgatroid99** [comment]
OK, the tests are green except for a couple of flakes.

**Pull request merged**